### ISSUE TITLE
`func host start` allows running in invalid directory

### ISSUE THREAD

**Issue by lastcoolnameleft** [Azure/azure-functions-core-tools#1489]
If you run `func host start` in a directory outside of the project, it exits with the error: `Unable to find project root.`.  This is good behavior.

However, if you run `func host start` in a function subdirectory (e.g. `HttpTrigger` from the examples), then it will create a new `host.json` file and start.  This is inconsistent behavior.  A preferred behavior would be to exit similar to if it was outside of the directory.

**Comment by snobu** [comment]
May i also suggest amending the original error message to:
```
Unable to find project root. Make sure you're running from the function app's root directory.
```

**Comment by brandonh-msft** [comment]
personally I'd appreciate a way to run `func host start --content <path to my function's root dir>` so I don't have to `cd` to the dir before running `func`

**Comment by elliotchaim** [comment]
I agree with @brandonh-msft . Not having this feature makes it difficult to work with functions projects in VS Code when working in a large repo. For example, we have `/dotnet/src/FunctionsApp`, but code needs to be open at `/`. This means we can't utilise `tasks.json` to run the functions app and speed up development

**Comment by unibeck** [comment]
Having a feature like @brandonh-msft mentioned would be much appreciated. Any update on if/when we could expect something like that?

**Comment by BoukeNijhuis** [comment]
Not totally sure if I am one the same page here, but I use the following command to prevent `cd`-ing:

`func host start --script-root <path to my function's root dir>`

So if you use `--script-root` instead of the proposed `--content` I think you are all set.

It does not solve this issue, but it helps the people who do not want to `cd`.

In case I misunderstood the request: my apologies for the confusion.
